conftest.py

```diff
@@ -10,3 +11,7 @@ def reset_environment_between_tests():
 
     os.environ.clear()
     os.environ.update(old_environ)
+
+    logger = logging.getLogger("appsignal")
+    for handler in logger.handlers:
+        logger.removeHandler(handler)
```

Comment:
I'd suggest moving this to a separate fixture so that it's not in the `reset_environment_between_tests` fixture. I'd also be okay with renaming the fixture to something more generic.

```suggestion

@pytest.fixture(scope="function", autouse=True)
def remove_logging_handlers_after_tests():
    yield
    
    logger = logging.getLogger("appsignal")
    for handler in logger.handlers:
        logger.removeHandler(handler)
```